app.js

```diff
@@ -83,17 +87,25 @@ const sessionOpts = Object.assign({
 
 app.use(session(sessionOpts));
 
+// check for cookies
+app.use(require('hof').middleware.cookies());
+
 // apps
 app.use(require('./apps/rtm/'));
 
 app.get('/cookies', (req, res) => res.render('cookies'));
 app.get('/terms-and-conditions', (req, res) => res.render('terms'));
 
+// 404's
+app.use(require('hof').middleware.notFound({
+  logger: require('./lib/logger'),
+  translate: i18n.translate.bind(i18n),
```

Comment:
Are we sure i18n has finished loading translations at this point? I needed to add an `i18n.on('ready', middleware)` for terms and cookies